WWDC/CursorShapeViewModifier.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  CursorShapeViewModifier.swift
+//  WWDC
+//
+//  Created by Allen Humphreys on 7/11/25.
+//  Copyright © 2025 Guilherme Rambo. All rights reserved.
+//
+
+import AppKit
+import SwiftUI
+
+/// Inside of a scroll view, the push/pop behaves not so great when re-capturing the hover after the scroll view
+/// scrolls. Scrolling the scroll view doesn't cancel the hover but it does change the cursor back to an arrow.
+///
+/// It may be possible to work around this via onContinuousHover and some cancelling delay shenanigans,
+/// but it seems like a lot of effort for a minor issue.
+struct CursorShapeViewModifier: ViewModifier {
```

Comment:
This is a neat modifier! I couldn't find where it's being used though. Am I missing something?

Reply:
@insidegui  I created this ahead of time to replace ActionLabel which is actually mostly used by the About window. ActionLabel is just text you can click and the pointing hand makes that clear. But there's one ActionLabel in the session summary... but I think it's broken so you maybe forgotten it's there 😂